modules/utils/docker/entrypoint_nats.sh

```diff
@@ -33,10 +33,31 @@ else
       sleep 1
   done
 
+  CBMA=$(extract_features_value "CBMA" $YAML_FILE)
```

Comment:
Hopefully we never use spaces or other problematic characters in filenames but just in case, to be safe it should be "$YAML_FILE" as in between double quotes :-)

Reply:
CBMA setting check will be removed once CBMA is default config.

Reply:
SECO-4104 added to backlog.